src/node.cc

```diff
@@ -1784,6 +1780,16 @@ inline int Start(uv_loop_t* event_loop,
   if (isolate == nullptr)
     return 12;  // Signal internal error.
 
+  if (per_process_opts->print_version) {
+    printf("%s\n", NODE_VERSION);
+    return 0;
+  }
+
+  if (per_process_opts->print_v8_help) {
+    V8::SetFlagsFromString("--help", 6);
+    return 0;
```

Comment:
This is unreachable code; `V8::SetFlagsFromString()` doesn't return when `--help` is present. Maybe change this to `UNREACHABLE()` for clarity?